examples/example.jsonld

```diff
@@ -0,0 +1,25 @@
+
+{
+  "@context": "http://pcdm.org/context-pcdm.json",
```

Comment:
You'll want to make sure the context value aligns with the name of the context file.